contracts/CompoundGovernor.sol

```diff
@@ -163,7 +163,11 @@ contract CompoundGovernor is
         uint256 _proposalId = hashProposal(targets, values, calldatas, descriptionHash);
         address _proposer = proposalProposer(_proposalId);
 
-        if (msg.sender != _proposer && msg.sender != proposalGuardian.account) {
+        // Check if the caller is the proposer, the proposal guardian, or if the proposalGuardian role is still valid.
```

Comment:
this isn't quite right -- we're checking if the caller is NOT the proposer or guardian, or if the guardian role is NOT valid... which, is kind of hard to parse logically. Is there any way to rewrite this condition so that it reads more intuitively?

Reply:
Ok, I've tried simplifying it here: https://github.com/ScopeLift/compound-governance-upgrade/pull/88/commits/fe756a76cba94b16400e40f827940171e8a2e800